### PR TITLE
Update lstchain_dl1ab.py

### PR DESCRIPTION
**Pull request by moralejo**
Fix issue introduced in #747

### CODE REVIEW

File: lstchain/scripts/lstchain_dl1ab.py [hunk 1]
```diff
@@ -203,13 +203,14 @@ def main():
         with tables.open_file(args.output_file, mode='a') as output:
             params = output.root[dl1_params_lstcam_key].read()
 
-            for tab in [output.root[dl1_params_lstcam_key], output.root[dl1_images_lstcam_key]]:
+            tabs = [output.root[dl1_params_lstcam_key]]
```
Comment (maxnoe) [inline lstchain/scripts/lstchain_dl1ab.py:206]: Why do we store this config in each of the tables? Why not just at the file root?

Comment (moralejo) [reply lstchain/scripts/lstchain_dl1ab.py:206]: Question for @Bultako I think... But we can just fix the problem here and leave that for another PR, right?

Comment (maxnoe) [reply lstchain/scripts/lstchain_dl1ab.py:206]: I think this is unnecessary duplication of the same info and this introduced this issue in the first place. `output.root.attrs` will always exist.

**Review by maxnoe — APPROVED** [review]

**Review by maxnoe — DISMISSED** [review]

File: lstchain/scripts/lstchain_dl1ab.py [hunk 2]
```diff
@@ -203,13 +203,14 @@ def main():
...
+            if not args.noimage:
+                tabs.append(output.root[dl1_images_lstcam_key])
+            for tab in tabs:
                 # need container to use lstchain.io.add_global_metadata and lstchain.io.add_config_metadata
                 for k, item in metadata.as_dict().items():
                     tab.attrs[k] = item
                 tab.attrs["config"] = str(config)
```
Comment (Bultako) [inline lstchain/scripts/lstchain_dl1ab.py:213]: I think that the image table is not produced during the execution of this script, but we are just copying the image table that has been produced with `lstchain_data_r0_to_dl1.py` script, so the image table has already its own provenance config stored in the execution of a previous script. So I would not reassign  the config to the image table. 

I would just **not consider** the image table, I would remove the loop in tabs and consider only the parameters table, as below:

```python
    # need container to use lstchain.io.add_global_metadata and lstchain.io.add_config_metadata
    for k, item in metadata.as_dict().items():
        output.root[dl1_params_lstcam_key].attrs[k] = item
        output.root[dl1_params_lstcam_key].attrs["config"] = str(config)
```

**Review by Bultako — COMMENTED** [review]
good catch @moralejo, I left some comments in between the lines.

File: lstchain/scripts/lstchain_dl1ab.py (hunk 1 above)
Comment (maxnoe) [reply lstchain/scripts/lstchain_dl1ab.py:206]: Letting the monitoring tables have their own provenance information at the node level does not mean not putting the information on how the hole file was create at the root.

Comment (Bultako) [reply lstchain/scripts/lstchain_dl1ab.py:206]: Sure.
In fact, the root level has already the global metadata info about software versions used in producing the **HDF5 file**, while the keys/tables have config and the software versions used in producing the **tables**

**Review by Bultako — APPROVED** [review]

**Pull request merged**